airbyte-integrations/connectors/source-commcare/source_commcare/source.py

```diff
@@ -34,15 +35,22 @@ def url_base(self) -> str:
     last_form_date = None
     schemas = {}
     unwantedfields = re.compile(r"^(case_|update_|meta|create_|commcare_).*$")
-
+  
     @property
     def dateformat(self):
         return "%Y-%m-%dT%H:%M:%S.%f"
 
     def scrubUnwantedFields(self, form):
-        newform = {k: v for k, v in form.items() if not self.unwantedfields.match(k)}
```

Comment:
Why are you removing the system excluding fields? Please keep this as is today.

Reply:
We don't need this. The idea is to not pull all the data from forms. and it was hard coded so I have made it dynamic and the user can choose which fields he doesn't want. It pulls a lot of unnecessary data.

Reply:
Yes, but the previous version is already removing the system fields. In your suggestion the user will must define ALL system fields to be removed.

Reply:
It will be an optional field. Either they can pull all the data or remove some of the fields. It works with both

Reply:
@marcosmarxm I have made some changes that you suggested. The idea is to excludes some form_fields and it can be optional
<img width="809" alt="Screenshot 2023-01-31 at 1 22 54 PM" src="https://user-images.githubusercontent.com/9019958/215699469-c90dd43d-927d-4a51-a55d-0251d08b828a.png">
here users can add the fields which they don't want to pull. So earlier it was hardcoded we've just made it dynamic.